ansible/roles/k8s_install/tasks/k8s_label_nodes.yml

```diff
@@ -35,6 +35,8 @@
   loop:
     - hosts: "{{ groups['openstack_control_plane'] }}"
       label: "openstack-control-plane=enabled"
+    - hosts: "{{ groups['openstack_control_plane'] }}"
+      label: "vault-storage=enabled"
```

Comment:
We can't have this by default. Vault is incompatible with our open-source license.

Reply:
ok removed the lines